src/plugins/inspector/public/adapters/request/request_adapter.ts

```diff
@@ -54,7 +54,7 @@ class RequestAdapter extends EventEmitter {
       name,
       startTime: Date.now(),
       status: RequestStatus.PENDING,
-      id: _.get(params, 'id', uuid()),
+      id: params.id ?? uuid(),
```

Comment:
:information_source: I just changed this line to see if it actually passes on CI too (and not just locally).